classes/UpgradeSelfCheck.php

```diff
@@ -32,6 +32,11 @@
 
 class UpgradeSelfCheck
 {
+    /**
+     * Recommended PHP Version. If below, display a notice.
+     */
+    const RECOMMENDED_PHP_VERSION = 70100;
```

Comment:
```suggestion
    const RECOMMENDED_PHP_VERSION = 70130;
```
Because of https://github.com/PrestaShop/PrestaShop/pull/13761/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R6

Reply:
So you mean PHP 7.1.30?

Reply:
Yep

Reply:
My bad, I mean `7.1.3`, was confused because 7.1.30 is the latest version of 7.1 :roll_eyes: